site/package.json

```
@@ -60,7 +60,7 @@
     "lodash": "^4.17.21",
     "md5": "^2.3.0",
     "mime-types": "^2.1.34",
-    "mock-block-dock": "0.0.21",
+    "mock-block-dock": "0.0.22",
```

`0.0.22` isn't published yet
```suggestion
    "mock-block-dock": "0.0.21",
```